app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ActiveRecord::Base
+  include Slugify::InstanceMethods
+  extend Slugify::ClassMethods
+
+  has_many :tweets
+
+  validates_presence_of :username, :email, :password
```

Comment:
You should be using [`has_secure_password`](http://api.rubyonrails.org/classes/ActiveModel/SecurePassword/ClassMethods.html#method-i-has_secure_password) to give users passwords and be able to authenticate against them.